src/parsers/multitest.js

```diff
@@ -1,29 +1,26 @@
-const PROTRACTOR_SHARDED = /------------------------------------/g
-const SPECFILE_REG = /.+Specs:\s(.*\.js)/g
+const PROTRACTOR_MULTITEST = /------------------------------------/g
+const SPECFILE_REG = /Specs:\s(.*\.js)/g
 
 export default {
-  name: 'ShardedParser',
+  name: 'MultiTestParser',
   test (output) {
-    return PROTRACTOR_SHARDED.test(output) && SPECFILE_REG.test(output)
+    return PROTRACTOR_MULTITEST.test(output) && SPECFILE_REG.test(output)
   },
 
   parse (failedSpecs, output) {
     let match = null
-    let testsOutput = output.split('------------------------------------')
-    testsOutput.shift()
-    let RESULT_REG = /\d+\sspec|assertions?,\s(\d+)\sfailures?/g
+    let testsOutput = output.split('------------------------------------').slice(1)
+    let RESULT_REG = /,\s0 failures/g
     testsOutput.forEach(function (test) {
       let specfile
       let result = 'failed'
       // retrieve specfile from run;
       while (match = SPECFILE_REG.exec(test)) { // eslint-disable-line no-cond-assign
         specfile = match[1]
       }
-      // check for string 'X specs, X failures' and verify that failures === 0;
+      // check for string '0 failures' and then marks the test as passed
       while (match = RESULT_REG.exec(test)) { // eslint-disable-line no-cond-assign
```

Comment:
I'll explain the intention for using slice, but both options are viable.

The logging contains something like:

```
Intro
----
log test 1
----
log test 2
conclusion.
```

Option A: Use slice.
This way you know that each item in testsOutput is a test, because Intro has been sliced out. If the specfile could not be retrieved in the test, an empty entry is added to failedSpecs.

Option B: check if (specfile && result=== failed)
This way you don't ensure that each item in testsOutput is a test. You just check if you can retrieve the failure and the specfile.

Personally I like option A better, because if a scenario exists where we can't determine the specfile using the current regexp, protractor-flake will re-run all tests. With option B it will just be ignored and skip the test (even if it failed).